textDisplay.cc

```diff
@@ -0,0 +1,82 @@
+#include "textdisplay.h"
+
+TextDisplay::TextDisplay(int n):View(n){
+	// Creates a new nxn textdisplay and sets the default value to '0'
+	theDisplay = new char*[n];
+	for (int row = 0; row < n; row++){
```

Comment:
OK sound good, I'll make that change. 
So then for the Load Game feature, should we do that within the game and then just notify the TextDisplay of each piece location in the saved game? Same idea?